core/sail/shacl/src/test/java/org/eclipse/rdf4j/sail/shacl/AbstractShaclTest.java

```diff
@@ -304,8 +304,8 @@ public static void afterClass() {
 				List<Object[]> collect = findTestCases(testCasePath, baseCase.name())
 						.stream()
 						.flatMap(path -> Stream
-//								.of(IsolationLevels.NONE, IsolationLevels.SNAPSHOT, IsolationLevels.SERIALIZABLE)// TODO: Uncomment this line!!!!!
-								.of(IsolationLevels.NONE)
+								.of(IsolationLevels.NONE, IsolationLevels.SNAPSHOT, IsolationLevels.SERIALIZABLE)
+//								.of(IsolationLevels.NONE)
```

Comment:
Re-enabled testing the ShaclSail against multiple isolation levels. This gives us over 13 000 test cases, but when executed in parallel on my ageing macbook it only take around 40 seconds.